src/elzero-task1/elzerotask1.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <style>
+      body {
+        font-family: 'Roboto', sans-serif;
+      }
+      span {
+        color: gray;
+      }
+      .red {
+        background-color: red;
+      }
+      .blue {
+        background-color: blue;
+      }
+      .green {
+        background-color: green;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>My info</h1>
+    <p>My Name Is: <b>Mohamed Khalifa</b></p>
+    <p>I'm <strong>Available</strong> For Hire</p>
+    <p>
+      My Hour Old Price Is <strike>$10</strike> And the New Price is
+      <mark>$30</mark>
+    </p>
+    <p>
+      Visit My Website From The Link
+      <a href="https://coffee.appsexplain.xyz/" target="_blank">
+        Wordpress Store</a
+      >
+    </p>
+    <hr />
+    <h2>Here Is Some Of My Clients</h2>
+    <span class="red">50 x 50</span>
+    <span class="blue">50 x 50</span>
+    <span class="green">50 x 50</span>
+    <hr />
+    <h2>My Skills</h2>
+    <ul>
+      <li>HTML</li>
+      <li>CSS</li>
+      <li>JavaScript</li>
+      <ul>
+        <li>Vuejs</li>
+        <li>Reactjs</li>
+        <li>Angular</li>
+
+        <ol>
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol

the ordered list starts from (4) , please fix it using the link above

Reply:
done , thank you for helping me